components/automate-cli/pkg/verifyserver/services/systemresourceservice/getosandfilesysteminfoutil.go

```diff
@@ -0,0 +1,88 @@
+package systemresourceservice
```

Comment:
Move this to /lib, so later others can use it. move test file also.

Reply:
Done